lib/gen-flake-outputs.nix

```diff
@@ -90,28 +129,8 @@ in
     };
     redistributable = build.buildDistTorchExtensions {
       inherit path doGetKernelCheck;
-      buildSets = buildSet;
+      bundleOnly = false;
       rev = revUnderscored;
     };
-    buildTree =
```

Comment:
This output has been broken for a while, so I don't think anyone uses it. IIRC it precedes shell support, it's more convenient to `nix develop` now and `build2cmake` there.